packages/devtools_app/lib/src/network/flutter/network_screen.dart

```diff
@@ -186,164 +186,169 @@ class NetworkScreenBodyState extends State<NetworkScreenBody> {
+                  ),
+                  numeric: true,
+                  onSort: (i, j) => _onSort(
+                    (HttpRequestData o) => o.status,
+                    i,
+                    j,
+                  ),
+                ),
+                DataColumn(
+                  label: const Text(
+                    'Duration (ms)',
+                    style: _headerTextStyle,
+                  ),
+                  numeric: true,
+                  onSort: (i, j) => _onSort(
+                    (HttpRequestData o) => o.durationMs,
+                    i,
+                    j,
+                  ),
+                ),
+                DataColumn(
+                  label: const Text(
+                    'Timestamp',
+                    style: _headerTextStyle,
+                  ),
+                  onSort: (i, j) => _onSort(
+                    (HttpRequestData o) => o.requestTime,
+                    i,
+                    j,
+                  ),
+                ),
+              ],
+              sortColumnIndex: _sortColumnIndex,
+              sortAscending: _sortAscending,
+            ),
+          );
+        },
+      ),
+    );
+  }
+
+  Widget _buildHttpProfilerBody(bool recording) {
+    return ValueListenableBuilder<HttpRequestData>(
+      valueListenable: dataTableSource.currentSelectionListenable,
+      builder: (context, HttpRequestData data, widget) {
+        return Expanded(
+          child: (!recording && dataTableSource.rowCount == 0)
+              ? Container(
+                  child: Center(
+                    child: recordingInfo(
+                      recording: recording,
+                      recordedObject: 'HTTP requests',
+                      isPause: true,
+                    ),
+                  ),
+                )
+              : Split(
+                  initialFirstFraction: 0.5,
+                  axis: Axis.horizontal,
+                  firstChild: (dataTableSource.rowCount == 0)
+                      ? Container(
+                          alignment: Alignment.center,
+                          child: const CircularProgressIndicator(),
+                        )
+                      : _buildHttpRequestTable(),
+                  // Only show the data page when there's data to display.
+                  secondChild: HttpRequestInspector(
+                    data,
+                  ),
+                ),
+        );
+      },
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
-    const double minIncludeTextWidth = 600;
     return ValueListenableBuilder<HttpRequests>(
       valueListenable: networkController.requestsNotifier,
-      builder: (context, data, widget) {
-        dataTableSource.data = data.requests;
-        return ValueListenableBuilder(
+      builder: (context, HttpRequests httpRequestProfile, widget) {
+        dataTableSource.data = httpRequestProfile.requests;
+        return ValueListenableBuilder<bool>(
```

Comment:
remove type

Reply:
If we remove both the type parameter this is inferred to be a `ValueListenableBuilder<dynamic>`. For the `bool` case that's probably fine, but for IDE autocompletion support we might want to keep it for the `ValueListenableBuilder<HttpRequests>`?